src/components/atoms/InternalCheckBox/InternalCheckBox.tsx

```diff
@@ -41,6 +42,8 @@ export type BaseCheckBoxProps = {
   onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
   onFocus?: (event: React.FocusEvent<HTMLInputElement>) => void;
   onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
+  "aria-label"?: string;
+  "aria-labelledby"?: string;
```

Comment:
Ideally the remaining input props should be being spread here. This isn't currently the case so i'm adding these to the white-list. I think we could make this component accept all the props of a vanilla checkbox going forward.